build_projects/dotnet-cli-build/PrepareTargets.cs

```diff
@@ -145,17 +140,94 @@ public static BuildTargetResult ExpectedBuildArtifacts(BuildTargetContext c)
             c.BuildContext["VersionBadge"] = Path.Combine(Dirs.Output, versionBadgeName);
 
             var cliVersion = c.BuildContext.Get<BuildVersion>("BuildVersion").NuGetVersion;
-            var sharedFrameworkVersion = c.BuildContext.Get<string>("SharedFrameworkNugetVersion");
-            var hostVersion = c.BuildContext.Get<HostVersion>("HostVersion").LockedHostVersion;
+            var sharedFrameworkVersion = DependencyVersions.SharedFrameworkVersion;
+            var hostVersion = DependencyVersions.SharedHostVersion;
 
+            // Generated Installers + Archives
             AddInstallerArtifactToContext(c, "dotnet-sdk", "Sdk", cliVersion);
-            AddInstallerArtifactToContext(c, "dotnet-host", "SharedHost", hostVersion);
-            AddInstallerArtifactToContext(c, "dotnet-sharedframework", "SharedFramework", sharedFrameworkVersion);
             AddInstallerArtifactToContext(c, "dotnet-dev", "CombinedFrameworkSDKHost", cliVersion);
-            AddInstallerArtifactToContext(c, "dotnet", "CombinedFrameworkHost", sharedFrameworkVersion);
             AddInstallerArtifactToContext(c, "dotnet-sharedframework-sdk", "CombinedFrameworkSDK", cliVersion);
             AddInstallerArtifactToContext(c, "dotnet-sdk-debug", "SdkSymbols", cliVersion);
 
+            //Downloaded Installers + Archives
+            AddInstallerArtifactToContext(c, "dotnet-host", "SharedHost", hostVersion);
+            AddInstallerArtifactToContext(c, "dotnet-sharedframework", "SharedFramework", sharedFrameworkVersion);
+            AddInstallerArtifactToContext(c, "dotnet", "CombinedFrameworkHost", sharedFrameworkVersion);
+
+            return c.Success();
+        }
+
+        [Target(
+            nameof(ExpectedBuildArtifacts), 
+            nameof(DownloadHostAndSharedFxArchives), 
+            nameof(DownloadHostAndSharedFxInstallers))]
+        public static BuildTargetResult DownloadHostAndSharedFxArtifacts(BuildTargetContext c) => c.Success();
+
+        [Target]
+        public static BuildTargetResult DownloadHostAndSharedFxArchives(BuildTargetContext c)
+        {
+            Mkdirp(Dirs.Packages);
```

Comment:
fixed as mentioned on other comment.

---
In reply to: [64623376](https://github.com/dotnet/cli/pull/3230#discussion_r64623376) [](ancestors = 64623376)